langchain4j-core/src/main/java/dev/langchain4j/retriever/Retriever.java

```diff
@@ -5,4 +5,8 @@
 public interface Retriever<T> {
 
     List<T> findRelevant(String text);
+
+    default List<T> findRelevant(Object memoryId,String text){
+       return findRelevant(text);
```

Comment:
Please throw an exception here: `throw new RuntimeException("Not implemented");`